string name
geometry_msgs/Point position
